test/functional/apps/discover/group1/_date_nanos.ts

```diff
@@ -15,30 +15,32 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'timePicker', 'discover']);
   const kibanaServer = getService('kibanaServer');
   const security = getService('security');
-  const fromTime = 'Sep 22, 2019 @ 20:31:44.000';
-  const toTime = 'Sep 23, 2019 @ 03:31:44.000';
+  const from = 'Sep 22, 2019 @ 20:31:44.000';
+  const to = 'Sep 23, 2019 @ 03:31:44.000';
 
   describe('date_nanos', function () {
     before(async function () {
       await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/date_nanos');
       await kibanaServer.savedObjects.clean({ types: ['search', 'index-pattern'] });
       await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/date_nanos');
+      await PageObjects.common.setTime({ from, to });
       await kibanaServer.uiSettings.replace({ defaultIndex: 'date-nanos' });
       await security.testUser.setRoles(['kibana_admin', 'kibana_date_nanos']);
       await PageObjects.common.navigateToApp('discover');
-      await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+      await PageObjects.timePicker.setAbsoluteRange(from, to);
```

Comment:
Since you are setting the time already in L26, I think setting it another time is not necessary?